sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command.py

```diff
@@ -170,15 +168,21 @@ def __init__(
         environment: Optional[Union[Environment, str]] = None,
         environment_variables: Optional[Dict] = None,
         resources: Optional[JobResourceConfiguration] = None,
-        services: Optional[Dict] = None,
+        services: Optional[
+            Optional[
+                Dict[
+                    str, Union[JobService, JupyterLabJobService, SshJobService, TensorBoardJobService, VsCodeJobService]
+                ]
+            ]
+        ] = None,
         queue_settings: Optional[QueueSettings] = None,
         **kwargs: Any,
     ) -> None:
         # validate init params are valid type
         validate_attribute_type(attrs_to_check=locals(), attr_type_map=self._attr_type_map())
 
         # resolve normal dict to dict[str, JobService]
-        services = _resolve_job_services(services)
+        services = _resolve_job_services(services)  # type: ignore[arg-type]
```

Comment:
Not sure what arg-type `_resolve_job_services` is requiring here, but using `cast` might be an option.

Reply:
fixed